src/cache/SortKeyCache.ts

```diff
@@ -10,9 +10,9 @@ export interface SortKeyCache<V> {
   getLessOrEqual(key: string, sortKey: string): Promise<SortKeyCacheResult<V> | null>;
 
   /**
-   * returns latest value stored for given contractTxId
+   * returns latest value stored for a given key
    */
-  getLast(contractTxId: string): Promise<SortKeyCacheResult<V> | null>;
+  getLast(key: string): Promise<SortKeyCacheResult<V> | null>;
 
   /**
    * returns last cached sort key - takes all contracts into account
```

Comment:
in general - since you're fixing the naming - please update all the comments in this interface (e.g. "takes all contracts" here)